src/API.jl

```diff
@@ -327,6 +327,9 @@ function up(ctx::Context, pkgs::Vector{PackageSpec};
             skip_writing_project::Bool=false,
             kwargs...)
     Context!(ctx; kwargs...)
+    if Operations.is_fully_pinned(ctx)
+        printpkgstyle(ctx.io, :Update, "All dependencies are pinned so cannot not be updated.", color = Base.warn_color())
```

Comment:
I wouldn't say that they cannot be updated (they could potentially be, but it was explicitly requested not to), but more that there's just nothing to do?

```suggestion
        printpkgstyle(ctx.io, :Update, "All dependencies are pinned - nothing to update.", color = Base.info_color())
```